docs/platform/documents/broadcast.md

```diff
@@ -8,4 +8,4 @@ Parameters:
 | **document**              | Document  | yes            | A valid [created document](/platform/documents/create.md) |
```

Comment:
Lets, describe documents object structure?

Reply:
It has been outdated, could you recheck to tell me if that is fine ?